src/app/clusters/scenes/scenes.cpp

```diff
@@ -65,6 +65,13 @@ uint8_t emberAfPluginScenesServerEntriesInUse = 0;
 EmberAfSceneTableEntry emberAfPluginScenesServerSceneTable[EMBER_AF_PLUGIN_SCENES_TABLE_SIZE];
 #endif
 
+static FabricIndex GetFabricIndex(app::CommandHandler * commandObj)
```

Comment:
This should be a utility of `CommandHandler` class. ReadHandler and WriteHandler already have `GetFabricIndex` that is initialized when the command exchange is initialized. The mFabricIndex could be initialized in `OnInvokeCommandRequest`.

Overall, this would allow replacing many bespokje `GetFabricIndex` that call `commandObj->GetExchangeContext()->GetSessionHandle().GetFabricIndex()` with a call of `commandObj->GetFabricIndex()`

Reply:
Agreed. I already [opened a ticked](https://github.com/project-chip/connectedhomeip/issues/11313) to do that.